src/store/server/history/types.ts

```diff
@@ -37,7 +37,4 @@ export interface ServerHistoryStateAllPrintStatusEntry {
         borderWidth: number
         borderRadius: number
     }
-    label: {
-        color: string
```

Comment:
why is this not used anymore?

Reply:
The charts are given an option object, which includes both data and styling options, in some places the styling was hard coded and not easy to modify according to theme. so I cut such properties out of the object & type to be able to dynamically pass in colour/styling.